### PR TITLE
Improved stats code/ increased test view size

### PR DESCRIPTION
**Pull request by marrero171**

### CODE REVIEW

**Review by abhinav3967 — APPROVED** [review]
Looks Good To Me What about @Technohacker

File: Characters/BaseFighter/BaseFighter.gd [hunk 1]
```diff
@@ -6,32 +6,33 @@ onready var body = $Body
...
-var strength : int
-var defence : int
 var is_grounded : bool = false
 
+export var stats = {
+	"speed" :  0, #make it 1-10
+	"strength" : 0, #same as above, make it 1-10
+	"defense" : 0,
+	"jump_velocity" : -620,
+	"min_jump_velocity" : -200 #reommended -200
```
Comment (Technohacker) [inline Characters/BaseFighter/BaseFighter.gd:22]: Typo :P

**Review by Technohacker — CHANGES_REQUESTED** [review]
Minor changes :)

File: Characters/BaseFighter/BaseFighter.gd (hunk 1 above)
Comment (abhinav3967) [reply Characters/BaseFighter/BaseFighter.gd:22]: Do you have a typo checker read script uhmm

**Pull request merged**